docs/volunteering/srawn.md

```diff
@@ -0,0 +1,79 @@
+# SR(A)WN
+
+The Student Robotics (Almost) Weekly Newsletter (SR(A)WN) is a semi-regular newsletter written by volunteers for volunteers.
+
+## Availablility
+
+SR(A)WN can be consumed electronically from several places:
+
+- Join the `#srawn` channel on [Slack](https://app.slack.com/client/T0EEPF1LH/C01GBT8NMSN) _recommended_
+- Join the `srawn` mailing list on [Google Groups](https://groups.google.com/g/srawn)
+- Read historical issues on the [SR(A)WN Archive](https://studentrobotics.org/srawn)
+- Subscribe to the [SR(A)WN RSS Feed](https://studentrobotics.org/srawn/rss.xml)
+
+The frequency of SR(A)WN issues is dependent on how much is going on within SR. In general there is an aim to wait until there is a good amount of interesting content.
+
+## Production Process
+
+The production process for SR(A)WN as of SR2021 follows the lifecycle in the following diagram:
+
+```mermaid
```

Comment:
I wonder if a numbered list may make more sense here, and be easier to consume? Albeit less "fancy".

Reply:
This was based on the precedent set by processes in other parts of the runbook. It may be worth creating a "recommended style" guide for the runbook.

Reply:
I think the thing here is that this process is strictly linear. We have a large number of other guides where linear step-by-step things are expressed as ordered lists. The only explicit diagrams I'm aware of are for processes which have several branches (and much more complex cycles) in them.

Was there a similar process you had in mind which made use of a diagram?